src/wallet/walletdb.cpp

```diff
@@ -418,6 +424,18 @@ ReadKeyValue(CWallet* pwallet, CDataStream& ssKey, CDataStream& ssValue,
                 strErr = "Error reading wallet database: Unknown non-tolerable wallet flags found";
                 return false;
             }
+        } else if (strType == "descriptor") {
+           unsigned int nID;
+           ssKey >> nID;
+           WalletDescriptor desc;
+           ssValue >> desc;
+           // TODO: figure out how to combine ssValue >> with MakeUnique<WalletDescriptor>
```

Comment:
@practicalswift e.g. this is the kind of thing I find feedback more useful on in the WIP stage